File: src/0771-Jewels-and-Stones/jewels_and_stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        ans: int = 0
        for s in stones:
            for j in jewels:
                if s == j:
                    ans += 1
        
        return ans


if __name__ == "__main__":
    jewels: str = "aA"
    stones: str = "aAAbbbb"

    solution = Solution()

    print(solution.numJewelsInStones(jewels, stones))
    

File: src/0771-Jewels-and-Stones/jewels_and_stones.cpp
#include <iostream>
#include <string>

class Solution {
public:
    int numJewelsInStones(std::string jewels, std::string stones) {
        int ans = 0;
        for (char s : stones) {
            for (char j : jewels) {
                if (s == j) {
                    ans += 1;
                }
            }
        }
        return ans;
    }
};

int main() {
    std::string jewels = "aA";
    std::string stones = "aAAbbbb";

    Solution solution;
    std::cout << solution.numJewelsInStones(jewels, stones);

    return 0;
}